### ISSUE TITLE
WebView2 appears transparent only on certain machines (only when included in a larger app)

### ISSUE THREAD

**Issue by cwmerkel** [MicrosoftEdge/WebView2Feedback#1368]
I'm using the the control in a WinForms application, and on a small subset of machines I've tested on the control appears to be transparent when displayed in the app.  The elements are there and can be clicked on (right click -> "open in new window" spawns a new browser window outside my app with the correct page).  I've constructed a number of simple sample apps and in those the webview2 control appears normal even on the problem machines, so there must be something about the larger app in which I am including it, but I have run out of ideas to try to diagnose what could be causing it.  All the main steps are being hit (initialization completed, NavStart, NavFinished, etc)

I know there isn't much to go on in this description,  I was wondering if there might be some known issue I'm not seeing or suggestions of what  could be causing it or to further diagnose the issue.

OS: Windows 10 Pro
Framework: .NET 4.7.2
package: 1.0.790-prerelease to 1.0.902-prerelease
Runtime: 91.0.864.41 (use evergreen distribution)

[AB#33900904](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/33900904)

**Comment by champnic** [comment]
Can you share how you are creating/parenting the control in the larger application? Is there anything special about that larger application compared to the sample apps (for example is AllowTransparency set)?

**Comment by cwmerkel** [comment]
I'll explain it some, but I want to highlight the main weirdness is that it works perfectly fine on five of the six machines I have tested it on, and it is only on one that it is appearing transparent (but that happens to be the head of QA's machine so it won't pass testing).

Basic application architecture is 
MainForm has a TableLayoutPanel.  Various UserControl derived classes are swapped in and out of that panel.
The particular UserControl class that shows the web view contains a single standard Panel to which the WebView2 control is added.

The sample app in this configuration works on the problem machine.  I have not been able to construct a "sample app" that fails on the problem machine in the same way as it does in the main app. 

There is a lot more that happens on the main app (logging into a server, syncing state).  It also references dozens of libraries, many 3rd party.

I've done a lot of debugging by MessageBox and see that all the same steps Initialization, NavStart, NavFinished steps occur on the machines where it works and the one where it doesn't. 

the AllowTransparency is not set on any forms

**Comment by champnic** [comment]
If you look at the sample app that works and the product app that's failing with [Spy++](https://docs.microsoft.com/en-us/visualstudio/debugger/introducing-spy-increment?view=vs-2019) to look at the HWND trees, are there any obvious differences?

**Comment by cwmerkel** [comment]
I used spy++ and simplified the tree of windows/controls even further.  Now the first and only thing the UserControl does is create and add a WebView2 control.  The only difference between the HWND trees that I found between our app and the sample app is that one of the intermediate windows has an addition WS_EX_COMPOSITED style set.

![spy](https://user-images.githubusercontent.com/8901082/122935865-aedbb700-d33e-11eb-9198-6f5df6e56df8.png)

This is from my development machine, where both apps work.  I can't install vsStudio on the offending machine.
